app/electron/main.ts

```diff
@@ -384,6 +374,12 @@ function startElecron() {
       mainWindow.webContents.send('appVersion', appVersion);
     });
 
+    mainWindow.webContents.on('will-navigate', (e, url) => {
+      if (url.startsWith('https://login.microsoftonline.com')) {
+        shell.openExternal(url);
+      }
+    });
```

Comment:
You already had a function adding this callback but not filtering the URL, should you remove the other one?